### ISSUE TITLE
Automatic Token Refresh for Embedded Use-Case

### ISSUE THREAD

**Issue by rasmusvan** [perses/perses#2178]
### Proposal

Regarding authorization for embedded panels: currently I must go to localhost:8080 to get a new token every 15 minutes to see the panels and data in my application. It would make sense to have background token refresh implemented for embedded use-case

https://matrix.to/#/!XNmgYIAndZOkEvQAbb:matrix.org/$BzHv47S9GqRg-JLuwb9MN91iCRE4ljBtIS009QZeNM8?via=matrix.org&via=gitter.im

**Comment by celian-garcia** [comment]
Yes I agree, the ts client (and also go client) should be better provided in authentication mechanism for people who want to use it from anywhere, not only embedded use case. 

I think the good name for this issue would be 
"Automatic refresh token in typescript client"

**Comment by Nexucis** [comment]
we can close this PR, it has been implemented.

Feel free to re-open it if you disagree
